test/unit/js/rotate.test.js

```diff
@@ -299,6 +300,33 @@ module("rotate: handler", {
   }
 });
 
+test("Check info object when rotate event fire.", function() {
+  // Given
+  var method = eg.invoke("rotate",[jQuery, null, this.fakeWindow, this.fakeDocument]);
+  var isVertical;
+  var isVertical2;
+  var isCall = false;
+  eg.isPortrait();
+  $(this.fakeWindow).on("rotate",function(e,info){
+    isCall = true;
+    isVertical = e.isVertical;
```

Comment:
I have same idea. but I cann't remove this code for compatibility. So. I wrote below commit msg.
```
fix(scrollend): Add info object when fire scrollend with.
Add jQuery style.
I can`t remove jindo style since version compatibility.

Ref #174
```